content/docs/preferences.md

```diff
@@ -332,34 +335,14 @@ A variety of settings that didn't fit elsewhere
     * `☐ Scroll to the last line after update`: will scroll to the end of the file after reloading from disk (otherwise, the cursor and scrolled-location stays where it was before the update)
 * `☐ Enable Notepad++ auto-updater`: will automatically download updates from the official website, once the development team has decided it's time to push an update to users.  If disabled, you will have to manually download the installer from the official website yourself.
 * `☐ Mute all sounds`: enable/disable sound feedback (example: if a search action in [**Find / Replace dialog**](../searching/#dialog-based-searching) results in the text not being encountered).
-* `☐ Set Save dialog file extension filter... for Normal Text`: for a file that has not been previously saved and the language of the file is set to Normal text, when this option is enabled, the **Save As** dialog's filter will default to `*.*` and require you to type an extension; when this option is disabled, the filter will default to `*.txt` and will auto-append the `.txt` extension if you don't type an extension or choose a different filter.  (Regardless of the state of this option, you can always select a different filter manually in the **Save As** dialog.)
+* `☐ Set Save dialog file extension filter... for Normal Text`: for a file that has not been previously saved and the language of the file is set to Normal text, when this option is enabled, the **Save As** dialog's filter will default to `*.*` and require you to type an extension; when this option is disabled, the filter will default to `*.txt` and will auto-append the `.txt` extension if you don't type an extension or choose a different filter.  (Regardless of the state of this option, you can always select a different filter manually in the **Save As** dialog.)  (This preference was removed in v7.9.6, and replaced by a [toggle in the **Save As** dialog](#other-toggles-and-settings).)
```

Comment:
Raises the interesting question of when such a bullet item is itself removed entirely from the user manual.

Reply:
As an example, when I moved "Sort Lines Randomly" in the menus, I didn't modify the user manual so that it remained in the old place with its old name and a note that says "(before v.8.0)".  But perhaps I should have done something like this?  Again, I'm not a proponent of all this "version" stuff hanging around (I like living in the here and now), but I'm not opposed to it either.

Reply:
Good point.  I usually remove the description in the old location when a setting has just moved, as long as it has a mention of where it used to be -- since I have that mention, I will remove the old description.

Reply:
@pryrt While you're polishing up things, if you'd like to add a mention that "Randomize Line Order" used to be "Sort Lines Randomly", I would certainly upvote that. :-)

Reply:
The "Randomize" is in editing.md, so it would be a separate issue / PR.  Which you could do if you wanted. ;-)

Reply:
I'm not that motivated about it.  I presumed you'd be back in that file for other reasons before too long. :-)